Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM php:7.3-apache
 
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
```

Comment:
Probably it makes sense to add a custom entrypoint.sh and configure it for the psalm-php service in the docker-compose.yml. The entrypoint could run composer install and then execute the Docker image's default entrypoint.